src/main/kotlin/org/radarbase/push/integration/GarminPushIntegrationResourceEnhancer.kt

```diff
@@ -1,6 +1,7 @@
 package org.radarbase.push.integration
 
 import com.fasterxml.jackson.databind.JsonNode
+import jakarta.inject.Singleton
```

Comment:
There still seems to be `javax.inject` usages for example in here: https://github.com/RADAR-base/RADAR-PushEndpoint/blob/bd4e6dbcb1919aebafe51406a365fad44a1e850f/src/main/kotlin/org/radarbase/push/integration/garmin/resource/GarminPushEndpoint.kt#L11
Should those be changed as well?

Reply:
Good point. Made the changes